bittensor/core/subtensor.py

```diff
@@ -223,6 +222,11 @@ def _get_substrate(self):
                     f"Connected to {self.network} network and {self.chain_endpoint}."
                 )
 
+            try:
+                self.substrate.websocket.settimeout(self._connection_timeout)
+            except (AttributeError, TypeError, socket.error, OSError) as e:
+                logging.warning(f"Error setting timeout: {e}")
+
         except ConnectionRefusedError:
             logging.error(
                 f"Could not connect to {self.network} network with {self.chain_endpoint} chain endpoint. Exiting...",
```

Comment:
You're gonna want to change this line since we're no longer exiting.

Reply:
> You're gonna want to change this line since we're no longer exiting.

log message is changed